bin/cv_regress

```diff
@@ -80,7 +80,11 @@ def is_test_indexed(project, name):
     for t in test_dirs:
         test_yaml = os.path.join(t, name, 'test.yaml')
         if os.path.exists(test_yaml):
-            test_spec = yaml.load(open(test_yaml))            
+            try:
```

Comment:
Is this a fix for the `YAMLLoadWarning: calling yaml.load() without Loader` warning?

Reply:
Yes, unfortunately some older PyYAMLs cannot handle supplying the Loader argument to .load, so it must be broken up as such.  I'll add a comment as otherwise it does look bizarre.

Reply:
Otherwise the newer PyYAMLs spit out a very ugly warning that litters our logs.